src/test_workflow/integ_test/service_opensearch.py

```diff
@@ -40,6 +40,7 @@ def __init__(
         self.dist = Distributions.get_distribution("opensearch", distribution, version, work_dir)
         self.dependency_installer = dependency_installer
         self.install_dir = self.dist.install_dir
+        self.password = 'myStrongPassword123!' if float('.'.join("3.0.0".split('.')[:2])) >= 2.12 else 'admin'
```

Comment:
should "3.0.0" be hardcoded here?

Reply:
Thanks, I was checking that the function worked and forgot to replace it with the actual version 🤦 . Fixed in [46cd35b](https://github.com/opensearch-project/opensearch-build/pull/4331/commits/46cd35bf3582d661e3753f90ca5f0075f9051960)